.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -30,21 +30,14 @@ body:
       label: "Way to reproduce:"
       description: >
         Provide a short, self-contained code example that reproduces the issue,
-        i.e., can be copy-pasted into the Python interpreter.
+        i.e., can be copy-pasted into the Python interpreter. Please also
+        include the full error message, traceback or output, if any.
       placeholder: |
+        ```python
         import numpy as np
-        import skimage
+        import skimage as ski
```

Comment:
does anybody outside of scikit-image use this.


Ok with the simplification otherwise.

Reply:
Not sure what you're asking, but I think since we added lazy loading this is becoming more and more common. `import skimage as ski; ski.filters.gaussian...`.

Reply:
i guess I find that "every package" wants a shorthand.

I feel like `skimage` is already a shorthand for `scikit-image`, so is `ski` a shorthand we are trying to establish.

It seems like it.

Reply:
There's a long history of this with `numpy as np`, `scipy as sp`, `matplotlib.pyplot as plt` etc., and we have to pick *something*. `ski` seems as good as any alternative.

Reply:
I'm not emotional about what kind of shorthand for `skimage` we use here; I think I have seen `ski` already used in some places but never `si`. Though I really like that `ski.module.func` makes it clear where a module comes from as opposed to some `filter.func`, `io.func` somewhere in the code.